policykit/policyengine/views.py

```diff
@@ -324,13 +325,14 @@ def initialize_policy(policy, action):
     from policyengine.models import Proposal, CommunityUser, BooleanVote, NumberVote
 
     users = CommunityUser.objects.filter(community=policy.community)
+    metagov = MetagovClient(policy, action)
 
     _locals = locals()
     _globals = globals()
 
-    wrapper_start = "def initialize(policy, action):\r\n"
+    wrapper_start = "def initialize(policy, action, metagov):\r\n"
```

Comment:
I noticed you expose metagov to all wrappers except for the filter wrapper -- is that intentional?

Reply:
Not really, I'll add it there as well.